### ISSUE TITLE
[Sugguestion] Colored Repeaters/Compareters

### ISSUE THREAD

**Issue by Yuna-She-They** [mrp-v2/MoreWires#3]
I've noticed that regular redstone repeaters/comparters, won't work are the colored Repeaters/Compaerters a part of a planned update?

**Comment by mrp-v2** [comment]
Can you rephrase that? I'm having a hard time understanding. I can tell you though that no, I don't plan to make colored Repeaters or Comparators.

**Comment by Yuna-She-They** [comment]
oh when ever I place a redstone repeater/compaerter down the signal doesn't activate any of the colored wires.

**Comment by mrp-v2** [comment]
Can you upload some pictures? Also what version are you on?

**Comment by Yuna-She-They** [comment]
1.16.4 just one morement while I get a screenshot

**Comment by mrp-v2** [comment]
Also what mod version.

**Comment by Yuna-She-They** [comment]
mod version: morewires-1.16.4-2.2.0.0-forge.jar screenshot: oh I jsut realized I built the  circut wrong, it's working fine
